### ISSUE TITLE
cfn generate fails with unhandled exception

### ISSUE THREAD

**Issue by gbernady** [aws-cloudformation/cloudformation-cli-go-plugin#118]
After upgrading the plugin to 0.1.3, when I call `cfn generate` I get the following output:

```
cfn generate                                                                                                                                                  master ✱
Change message for Go plugin v0.1.3:
Generated models no longer use the types exported in the encoding package.
Your model's fields have been regenerated using standard pointer types (*string, *int, etc) as used in the AWS Go SDK.
The AWS SDK has helper functions that you can use to get and set your model's values.

Make the following changes to your handler code as needed:

* Replace `encoding.New{Type}` with `aws.{Type}`
* Replace `model.{field}.Value()` with `aws.{Type}Value(model.{field})`

Where {Type} is either String, Bool, Int, or Float64 and {field} is any field within your generated model.

=== Unhandled exception ===
Please report this issue to the team.
Issue tracker: https://github.com/aws-cloudformation/aws-cloudformation-rpdk/issues
Please include the log file 'rpdk.log'
```

The `rpdk.log` looks like this:

```
[2020-02-25T08:12:39Z] DEBUG    - Logging set up successfully
[2020-02-25T08:12:39Z] DEBUG    - Running generate: Namespace(command=<function generate at 0x110e1ae60>, subparser_name='generate', verbose=0, version=False)
[2020-02-25T08:12:39Z] DEBUG    - Root directory: /Users/berng/Projects/pega/berng/go-cfn-resource-provider
[2020-02-25T08:12:39Z] DEBUG    - Loading project file '/Users/berng/Projects/pega/berng/go-cfn-resource-provider/.rpdk-config'
[2020-02-25T08:12:39Z] INFO     - Validating your resource specification...
[2020-02-25T08:12:39Z] DEBUG    - Rewriting refs in '<BASE>' (file:///Users/berng/Projects/pega/berng/go-cfn-resource-provider/pega-service-goresource.json)
[2020-02-25T08:12:39Z] DEBUG    - Writing Resource Role CloudFormation template: /Users/berng/Projects/pega/berng/go-cfn-resource-provider/resource-role.yaml
[2020-02-25T08:12:39Z] DEBUG    - Overwriting '/Users/berng/Projects/pega/berng/go-cfn-resource-provider/resource-role.yaml'
[2020-02-25T08:12:39Z] DEBUG    - Generate started
[2020-02-25T08:12:39Z] DEBUG    - Writing Types
[2020-02-25T08:12:39Z] DEBUG    - Overwriting '/Users/berng/Projects/pega/berng/go-cfn-resource-provider/cmd/resource/model.go'
[2020-02-25T08:12:39Z] DEBUG    - Writing project: /Users/berng/Projects/pega/berng/go-cfn-resource-provider/cmd/main.go
[2020-02-25T08:12:39Z] DEBUG    - Overwriting '/Users/berng/Projects/pega/berng/go-cfn-resource-provider/cmd/main.go'
[2020-02-25T08:12:39Z] DEBUG    - Writing Makefile: /Users/berng/Projects/pega/berng/go-cfn-resource-provider/Makefile
[2020-02-25T08:12:39Z] DEBUG    - Overwriting '/Users/berng/Projects/pega/berng/go-cfn-resource-provider/Makefile'
[2020-02-25T08:12:40Z] DEBUG    - protocolVersion version change from None to 1.0
[2020-02-25T08:12:40Z] DEBUG    - pluginVersion version change from None to 0.1.3
[2020-02-25T08:12:40Z] DEBUG    - Unhandled exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/cli.py", line 98, in main
    args.command(args)
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/generate.py", line 15, in generate
    project.generate()
  File "/usr/local/lib/python3.7/site-packages/rpdk/core/project.py", line 271, in generate
    return self._plugin.generate(self)
  File "/usr/local/lib/python3.7/site-packages/rpdk/go/codegen.py", line 200, in generate
    project._write_settings(LANGUAGE)
AttributeError: 'Project' object has no attribute '_write_settings'
```

I dived into cloudformation-cli implementation and it seems the `_write_settings` function was redefined in PR [#386](https://github.com/aws-cloudformation/cloudformation-cli/pull/386).

**Comment by sayboras** [comment]
I just tried to upgrade as well, and have exactly the same issue even after changing go code to make it compilable.

**Comment by gbernady** [comment]
I changed the line 200 in `/usr/local/lib/python3.7/site-packages/rpdk/go/codegen.py` locally on my machine from `project._write_settings(LANGUAGE)` to `project.write_settings()` and it seems to fix the issue.

**Comment by sayboras** [comment]
Awesome, thanks for sharing. Hope that it will be fixed soon.

**Comment by jotompki** [comment]
Ah looks like the go plugin was relying on a method removed in the latest release of the CLI. Will prep a pull request to use the new version.

**Comment by gbernady** [comment]
I can confirm the issue is fixed in [v0.1.4](https://github.com/aws-cloudformation/cloudformation-cli-go-plugin/releases/tag/v0.1.4). Thanks! Closing the issue.
